flask_restplus/api.py

```diff
@@ -244,7 +249,7 @@ def _register_apidoc(self, app):
         conf['apidoc_registered'] = True
 
     def _register_specs(self, app_or_blueprint):
-        if self._add_specs:
+        if self._add_specs and not self._hide_specs_url:
```

Comment:
So this can become

```
if self._add_specs:
   self.specs(self._spec_view)
```

or something like that.

Reply:
modified so it just added the url route